net/iperf/Makefile

```diff
@@ -45,6 +45,7 @@ define Package/iperf/config
 endef
 
 CONFIGURE_ARGS += \
+          --enable-fastsampling \
```

Comment:
why is whitespace different here?

Reply:
Sorry @neheb, don't understand your question. What whitespace difference?